### DISCUSSION TITLE
Possible to clone ALL branches for each repo?

### DISCUSSION THREAD

**Discussion by jaketoolson** [gabrie30/ghorg#165] [Q&A] [answered]
Curious if it's possible through a flag, to also pull all branches for each repo.

**Comment by gabrie30** [comment]
There could be a flag that would perform a `git fetch --all` on each repo, is that what you were thinking?

**Comment by gabrie30** [comment, marked as answer]
https://github.com/gabrie30/ghorg/issues/168

**Reply by jaketoolson** [reply]
"That was easy" haha. Thanks.
